cmd/logs.go

```diff
@@ -371,3 +461,158 @@ func logsDNS() {
 func logsIngress() {
 	logsTerraform("ingress.tf-job")
 }
+
+type logFlags struct {
+	sinceSeconds  time.Duration
+	sinceTime     string
+	tail          int64
+	elasticsearch bool
+}
+
+func newLogsFlags() *logFlags {
+	return &logFlags{
+		tail: -1,
+	}
+}
+
+type logResponse struct {
+	Hits struct {
+		Hits []struct {
+			Source struct {
+				Timestamp time.Time `json:"@timestamp"`
+				Log       string    `json:"log"`
+				Severity  string    `json:"severity"`
+				Source    string    `json:"source"`
+			} `json:"_source"`
+		} `json:"hits"`
+	} `json:"hits"`
+}
+
+type rng struct {
```

Comment:
You could give a better name. Maybe `range` is a better name (if rng means range in your context)?

Reply:
range is a keyword of golang and therefore not usable

Reply:
Yup, but the problem is that `range` is reserved keyword in Golang